### DISCUSSION TITLE
all notes feature

### DISCUSSION THREAD

**Discussion by Z1xus** [Zrips/CMI#6139] [Enhancement]
Please add the ability to check notes of all player at one command, it would be very useful for moderators.

**Comment by CloudeLecaw** [comment]
And very painfull if u have to read the notes of 1000 players that ate offline just tonhet the  notes of one.

+ You have this setting
```
Notes:
  # When enebled, when player logs in who has alert set on him, staff member will get notification that this player have some notes attached to him
  ShowOnAlertEvent: true
```
